patches/capnp-es.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/dist/compiler/main.mjs b/dist/compiler/main.mjs
+index 6999c0e0c50570efd51959661ffc611f561bd31b..c37f97d5d786e4fc316acd4f5780e8e3976b21dd 100755
+--- a/dist/compiler/main.mjs
++++ b/dist/compiler/main.mjs
+@@ -38,7 +38,7 @@ Options:
+ `;
+ const sources = [];
+ const options = [];
+-let outFormats = ["js"];
++let outFormats = ["ts"];
```

Comment:
Would you mind creating/linking to an upstream issue? Is it possible to get rid of this patch eventually?

Reply:
Yeah, I'm going to put up a PR today